app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java

```diff
@@ -132,6 +141,11 @@ public String getCurrentUsername() {
         return mPref.getString("username", "null");
     }
 
+    public String getCurrentSchoolId()
+    {
+        return mPref.getString("currentSchool", "null");
```

Comment:
Why `"null"` as default value and not a real `null`? This way you get a string with the value `null`.